Tasks/KubernetesManifestV0/src/utils/utilities.ts

```diff
@@ -103,4 +68,42 @@ export function annotateChildPods(kubectl: Kubectl, resourceType, resourceName,
     }
 
     return commandExecutionResults;
-}
\ No newline at end of file
+}
+
+export function replaceAllTokens(currentString: string, replaceToken, replaceValue) {
+    let i = currentString.indexOf(replaceToken);
+    if (i < 0) {
+        tl.debug(`No occurence of replacement token: ${replaceToken} found`);
+        return currentString;
+    }
+
+    let newString = currentString.substring(0, i);
+    let leftOverString = currentString.substring(i);
+    newString += replaceValue + leftOverString.substring(Math.min(leftOverString.indexOf("\n"), leftOverString.indexOf("\"")));
+    if (newString == currentString) {
+        tl.debug(`All occurences replaced`);
+        return newString;
+    }
+    return replaceAllTokens(newString, replaceToken, replaceValue);
+}
+
+export function isEqual(str1: string, str2: string, ignoreCase: boolean): boolean {
```

Comment:
Instead of ignoreCase being a bool you could use an enum instead to identify the comparer logic?